homework/src/index.js

```diff
@@ -30,18 +42,141 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { html: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
-      }
+  //this is an object creator
+  function RepositoryInfo(sourceObj) {
+    this.Description = sourceObj.description;
+    this.Forks = sourceObj.forks;
+    const dateOfUpdate = new Date(sourceObj.updated_at);
+    this.Updated = dateOfUpdate.toLocaleString();
```

Comment:
Object property names should be in camelCase.

Reply:
Actually, you are right, I named them with starting capital letters because I used them in the web page. Here is the example: `Forks: 5`